server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -483,6 +487,24 @@ public void moveSegment(
     }
   }
 
+  /**
+   * This method should be used for only testing.
```

Comment:
nit: comment is not needed since we already have @VisibleForTesting annotation

Reply:
addressed